lib/mocha-tests-lint.ts

```diff
@@ -34,7 +34,7 @@ export async function lintMochaTests(
 	try {
 		await Promise.all(allCheckPromises);
 		return new MochaListResult(errorsFound ? message : 'OK', errorsFound);
-	} catch (e) {
+	} catch (e: any) {
```

Comment:
This is the Typescript 4.4 thing I reckon: [Defaulting to the unknown Type in Catch Variables](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables)

Reply:
Yup, it will be